### PR TITLE
[C-API] Implemented WasmEdge_ImportObjectGetModuleName.

### PR DESCRIPTION
**Pull request by apepkuss**
Signed-off-by: Xin Liu <sam@secondstate.io>

### CODE REVIEW

**Comment by apepkuss** [comment]
Hi @q82419, please help review this PR. Thanks a lot!

**Comment by apepkuss** [comment]
Please notice the test on Fedora 35 is timeout. That should not be caused by this PR.

```bash
16/21 Test #16: wasiTests ........................***Timeout 1500.10 sec
```

File: include/api/wasmedge/wasmedge.h [hunk 1]
```diff
@@ -2772,6 +2772,14 @@ WasmEdge_ImportObjectCreateWasmEdgeProcess(const char *const *AllowedCmds,
                                            const uint32_t CmdsLen,
                                            const bool AllowAll);
 
+/// Get the module name.
+///
```
Comment (q82419) [inline include/api/wasmedge/wasmedge.h:2776]: Please add the comment.

```
/// The returned string object is linked to the module name of the import object context, and the caller should __NOT__ call the `WasmEdge_StringDelete`.
```

**Review by q82419 — CHANGES_REQUESTED** [review]

**Comment by q82419** [comment]
> Please notice the test on Fedora 35 is timeout. That should not be caused by this PR.
> 
> ```shell
> 16/21 Test #16: wasiTests ........................***Timeout 1500.10 sec
> ```

It's known issue. You can re-run the CI or ignore this. Thanks.

File: include/api/wasmedge/wasmedge.h (hunk 1 above)
Comment (apepkuss) [reply include/api/wasmedge/wasmedge.h:2776]: Accepted. I'll Add the doc comments. Thanks a lot!

Comment (apepkuss) [reply include/api/wasmedge/wasmedge.h:2776]: The issue has been fixed. Thanks!

**Review by q82419 — APPROVED** [review]

**Comment by apepkuss** [comment]
@q82419 Thanks for the review!

**Pull request merged**